hoodie-client/src/main/java/com/uber/hoodie/config/HoodieHBaseIndexConfig.java

```diff
@@ -69,6 +76,34 @@
    */
   public static final float DEFAULT_HBASE_QPS_FRACTION = 0.5f;
 
+  /**
+   *  Property to decide if HBASE_QPS_FRACTION_PROP is dynamically calculated based on volume
```

Comment:
Please open a new PR for updating documentation in https://hudi.incubator.apache.org/configurations.html. That PR should be against the branch  "asf-site".

Reply:
Will do, thanks